django/contrib/auth/management/commands/createsuperuser.py

```diff
@@ -20,6 +20,7 @@ class NotRunningInTTYException(Exception):
 class Command(BaseCommand):
     help = 'Used to create a superuser.'
     requires_migrations_checks = True
+    private_options = ('username', )
```

Comment:
This seems suspicious -- if it's needed it should probably be added by `for field in self.UserModel.REQUIRED_FIELDS`. What's failing without it?